mastercard.php

```diff
@@ -871,7 +872,7 @@ protected function postProcess()
                 'mpgs_api_password',
                 'test_mpgs_api_password',
                 'mpgs_webhook_secret',
-                'test_mpgs_webhook_secret'
+                'test_mpgs_webhook_secret',
```

Comment:
Extraneous comma?

Reply:
According to the following PHP specification - https://www.php.net/manual/en/function.array.php.

>Having a trailing comma after the last defined array entry, while unusual, is a valid syntax.

It's commonly used for convenience so we can easily add another element to the array without adding the trailing comma to the last entry. 
As soon as it's allowed by language syntax and commonly used, IDE adds this comma automatically when I use the autoformatting tool because it's a part of PSR-2 recommendations for the IDE (https://devdocs.prestashop.com/1.7/development/coding-standards/#php-code-conventions). So, this particular comma is added automatically to make the code style unified across the module code. 

@ellenchristine Please let me know if it violates some code styling guidelines you use. I can configure my IDE to stick to it to avoid future violations.

Reply:
Gotcha, I didn't realize it was valid in PHP. I'll disregard. :)